prov/sockets/src/sock_conn.c

```diff
@@ -493,6 +494,8 @@ int sock_ep_connect(struct sock_ep_attr *ep_attr, fi_addr_t index,
 	socklen_t lon;
 	int valopt = 0;
 	struct pollfd poll_fd;
+	char buf[256] = {0};
+	size_t buf_len = 256;
```

Comment:
Please drop the initialization of buf.  buf_len can be set to sizeof(buf)

Reply:
Dropped and use ofi_straddr_log() as you suggested.